content/en/tracing/setup/dotnet.md

```diff
@@ -144,15 +144,16 @@ COR_ENABLE_PROFILING=1
 COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
 ```
 
-`COR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and `DD_INTEGRATIONS` is set globally for all processes.
+`COR_PROFILER_PATH` is not required because the MSI installer registers the native COM library's path in the Windows Registry, and environment variables `DD_INTEGRATIONS` and `DD_DOTNET_TRACER_HOME` are set globally for all processes.
 
-If you did not use the MSI installer, set all four environment variables:
+If you did not use the MSI installer, set all five environment variables:
```

Comment:
```suggestion
If you did not use the MSI installer, set the following environment variables:
```

Reply:
I think @zacharycmontoya's intent was to emphasize that in this specific case, they need to set _all five_ env vars instead of just two like mentioned a few paragraphs above.

Reply:
Ok. We can skip changing this if it's important to specify. I usually try to eliminate saying "follow these # of steps" because when things get added or removed, it's easy to forget to update the number.

Reply:
Yes, for the two lines starting with `If you did not use the MSI installer`, I would like to keep the specific number included to avoid any ambiguity. I am happy to change it for the third suggestion on the Linux setup.